bng.py

```diff
@@ -89,9 +89,9 @@ def to_osgb36(coords):
         except IndexError:
             raise ValueError('Invalid 100km grid square code')
         nDigits = (len(coords) - 2) / 2
```

Comment:
If we cast to integer here e.g. `nDigits = int(len(coords) -2 / 2)`, then it saves doing it lots of times below.

Reply:
I agree. That makes more sense.